Makefile

```diff
@@ -7,7 +7,7 @@ ifneq "$(wildcard /usr/local/opt/openssl/lib)" ""
 	LDFLAGS += -L/usr/local/opt/openssl/lib
 endif
 
-PIP = LDFLAGS="$(LDFLAGS)" pip
+PIP = LDFLAGS="$(LDFLAGS)" pip -q
```

Comment:
I'd remove this then too, and purely rely on `PIP_QUIET` that you're setting as the env var. I'd argue, if someone wnats this locally or whatever, they can also set the env var.